default_bus_layout.tres

```diff
@@ -1,3 +1,26 @@
-[gd_resource type="AudioBusLayout" format=2]
+[gd_resource type="AudioBusLayout" load_steps=2 format=2]
+
+[sub_resource type="AudioEffectReverb" id=1]
+resource_name = "Reverb"
+room_size = 0.0
+damping = 0.0
+wet = 0.1
 
 [resource]
+bus/0/volume_db = -0.130497
```

Comment:
Are these volume levels deliberately set down by a tenth of a decibel, or was this the fault of the editor?